pkg/domain/accounts/usecases.go

```diff
@@ -0,0 +1,29 @@
+package accounts
+
+import (
+	"errors"
+
+	"github.com/daniel1sender/Desafio-API/pkg/domain/entities"
+)
+
+var (
+	ErrExistingCPF = errors.New("cpf informed alredy exists")
+	ErrIDNotFound  = errors.New("account id isn't found")
```

Comment:
a little typo here, isn't it?
```suggestion
	ErrAccountNotFound  = errors.New("account not found")
```

Reply:
Thank you for this suggestion!